docs/cli.md

```diff
@@ -208,6 +210,56 @@ Run data diagnosis and output the results in jsonl format:
 sb result diagnosis --data-file outputs/results-summary.jsonl --rule-file rule.yaml --baseline-file baseline.json --output-file-format json
 ```
 
+Run data diagnosis and output the results in markdown format with 2 valid decimal places:
+```bash title="SB CLI"
+sb result diagnosis --data-file outputs/results-summary.jsonl --rule-file rule.yaml --baseline-file baseline.json --output-file-format md --decimal-place-value 2
+```
+
+### `sb result summary`
```

Comment:
do you need to add a tutorial in document like [data diagnosis](https://microsoft.github.io/superbenchmark/docs/user-tutorial/data-diagnosis)?

Reply:
sure, I plan to submit another pr for that